### PR TITLE
[issue 2279] Update how we generate ERD diagrams for the database schema

### PR DESCRIPTION
**Pull request by babebe**
## Summary
Fixes #{[2279](https://github.com/HHS/simpler-grants-gov/issues/2279#issue-2557376031)}

## Changes proposed
 Replace sadisplay library with eralchemy to generate ERDs

## Context for reviewers
Updated existing script `create_erds` to use the new library for generating ERDs. Newly generated diagrams(included)  contain all the table and columns in the database. 

## Additional information
Screenshots, GIF demos, code examples or output to help show the changes working as expected.

### CODE REVIEW

File: api/bin/create_erds.py
Comment (chouinar) [inline api/bin/create_erds.py:6]: I'm not sure I like `sqlalchemy_schemadisplay` - it changes quite a bit in our ERDs that just makes them incorrect. It doesn't look like it picks up foreign keys for a large percentage of our tables, and our non-foreign key relationships that we wouldn't want in an ERD (they're not actually in the DB, just an ORM-level thing).

Did you get a chance to test out eralchemy? From my 5 minutes of testing it seemed a bit closer to what we currently have with pretty minimal configuration (just give it the metadata object)

File: documentation/api/database/erds/transfer-schema.png
Comment (chouinar) [inline documentation/api/database/erds/transfer-schema.png:1]: While we're at it, we should just not generate this file anymore. We'll delete that table soon.

File: documentation/api/database/erds/staging-schema.png
Comment (chouinar) [inline documentation/api/database/erds/staging-schema.png:1]: Since this looks.. odd - I was curious if there was a way to adjust the formatting. Looks like we'd have to adjust the way eralchemy uses graphviz to do that: https://github.com/eralchemy/eralchemy/issues/63

Not in the scope of this PR, but just noting it down.

File: documentation/api/database/erds/transfer-schema.png
Comment (chouinar) [inline documentation/api/database/erds/transfer-schema.png:1]: Could you also remove the reference to this file from https://github.com/HHS/simpler-grants-gov/tree/main/documentation/api/database/erds

File: documentation/api/database/erds/full-schema.png
Comment (chouinar) [inline documentation/api/database/erds/full-schema.png:1]: I wonder how helpful this full schema is as well. @mdragon - I'll defer to you whether you think this is as helpful as the two separate api/staging schema diagrams we have.

**Review by chouinar — COMMENTED** [review]
Looks good, just a few minor details / want to sort out whether to keep the full schema.

File: documentation/api/database/erds/full-schema.png
Comment (mdragon) [reply documentation/api/database/erds/full-schema.png:1]: I think it's probably not as useful and in the Readme we can just make sure that we spell out they're in the same DB and we just have them diagramed by schema for ease of viewing.

File: documentation/api/database/erds/transfer-schema.png
Comment (babebe) [reply documentation/api/database/erds/transfer-schema.png:1]: done

File: documentation/api/database/erds/full-schema.png
Comment (babebe) [reply documentation/api/database/erds/full-schema.png:1]: Removed full schema, cleaned up code and  updated the readme

**Comment by chouinar** [comment]
@babebe - could you run a `poetry lock --no-update` - poetry complains for me when I try to install this branch locally (otherwise looks good and can approve once that's done)

**Comment by babebe** [comment]
> @babebe - could you run a `poetry lock --no-update` - poetry complains for me when I try to install this branch locally (otherwise looks good and can approve once that's done)

@chouinar should be good now

**Review by chouinar — APPROVED** [review]
LGTM!

**Pull request merged**
